### PR TITLE
Lowell Standardizer

### PR DESCRIPTION
**Pull request by arihani2**

### CODE REVIEW

**Comment by arihani2** [comment]
[Lowell Discovery Telescope Standardizer](https://trello.com/c/r9P4LviY/52-lowell-discovery-telescope-standardizer)

**Review by mrawls — APPROVED** [review]
This looks excellent! Great job on the docstrings in particular. One question, I see you added a test image and some standardized values to the tests directory, but I don't think I see anywhere that a test is actually run. Is that happening elsewhere/later?

**Comment by arihani2** [comment]
> I don't think I see anywhere that a test is actually run. Is that happening elsewhere/later?

The tests run automatically when you add the cutout file to the upload/data/tests directory and when you add the expected values to the yaml file. The tests go through all the files in the directory so no new test needs to be written.

File: trail/upload/process_uploads/standardizers/lowell_standardizer.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+
```
Comment (DinoBektesevic) [inline trail/upload/process_uploads/standardizers/lowell_standardizer.py]: No need for this newline

**Review by DinoBektesevic — APPROVED** [review]
Looks good!

**Pull request merged**